src/components/Shared/ConfirmDialog.jsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+// components
+import { Button, Dialog, DialogTitle, DialogActions } from '@mui/material';
+
+export default function ConfirmDialog(props) {
```

Comment:
rename this component to something like "DeleteTagConfirmDialog" since it is specific for tags currently.